recipe/meta.yaml

```diff
@@ -1,4 +1,5 @@
-{% set version = "8.1.0" %}
+{% set sha = "3056e46" %}
+{% set version = "8.1.0." ~ sha %}
```

Comment:
@danlipsa what does this do?

Reply:
Appends the sha to the file name so that you know which VTK you have.